core/src/test/java/org/hibernate/ogm/backendtck/type/BuiltInTypeTest.java

```diff
@@ -185,6 +193,7 @@ public void testStringAsLobSupport() throws Exception {
 		assertEquals( "Original and loaded data do not match!", text, loadedBookmark.getLobWithString() );
 	}
 
+	@SkipByGridDialect(value = { GridDialectType.ORIENTDB }, comment = "Byte array not supported in JSON")
```

Comment:
Problem is that OrientDB query parser is incorrect parse JSON escaping.
______________________________________
aa [main] 14:33:07,128 DEBUG org.hibernate.datastore.ogm.orientdb.OrientDBDialect: insertOrUpdateTuple:Key: id  ( bcf92040-8813-488c-92a7-422cff70dafa ). Query: insert into Book
mark content {"data":"eLuLeqkmovJI1a7WR2OTvVAbLowD\/N4WEHJ6\/2UMmTQNeNSeSE87fZvMvfFStfLYpA0PFGmZ\/m+QmyJ4PPdVm63qoSAW+\/cY1DvmrJ2H4fYhkjasKzUmcOuxhiPUpzYfZXXfAGk0AgqX3fOZ8XkBeoe
KyhFpJDojxcMtG9BO8WjGgpDizD0WgqkSY7hoQZ+PV2Vig7fDn3MnRDGZZLPyyWaSL3J+5DWPdALaQfN+eAhacvqJnnHDPf2yV5Xc8RpqXVUyEPGig60=","isRead":"N","id":"bcf92040-8813-488c-92a7-422cff70dafa","
isPrivate":"F","isShared":0}
______________________
But ... other base64 parsed ...  I created issue to OrientDB team. They promise to fix.